core/tauri/src/webview/plugin.rs

```diff
@@ -159,6 +159,20 @@ mod desktop_commands {
   setter!(set_webview_position, set_position, Position);
   setter!(set_webview_focus, set_focus);
 
+  #[cfg(any(debug_assertions, feature = "devtools"))]
```

Comment:
any reason why this is only enabled on these features

Reply:
copypasta lol